inst/templates/run-tutorial.R

```diff
@@ -0,0 +1,5 @@
+#' Run a tutorial
+#' @keywords NULL
+#' @export
+#' @name run_tutorial
+learnr::run_tutorial
```

Comment:
The form of this seems a bit surprising to me. Have you created a test package and used `use_tutorial(name = "aaa", title = "bbb", learner_friendly = TRUE)`, run `document()`, installed it and checked that it does what you intend?

If you look at the template added by `use_tidy_eval()` and the way devtools re-exports functions from other packages, there are some differences. Based on those, I would expect to see an `@importFrom` directive that imports `run_tutorial` from learnr.

What is the intent with `@keywords NULL`?

Reply:
@jennybc I copied this bit from @lionel-.  This is supposed to create a documentation page for each re-exported function (in this case just one) that links to the original function.

https://github.com/tidyverse/purrr/blob/master/R/reexport-rlang.R

Reply:
Ok so I guess these are two different styles of exposing a function from package B in package A.

That makes my question/comment: are you sure you want this style as opposed to the one used by other re-export helpers in usethis? I don't have an opinion here, just want to know that you like the end result.